src/components/ha-cover-controls.html

```diff
@@ -0,0 +1,60 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+
+<link rel="import" href="../util/cover-model.html">
+
+<dom-module id="ha-cover-controls">
+  <template>
+    <style>
+      .state {
+        text-align: right;
+        white-space: nowrap;
+        width: 127px;
```

Comment:
Are you sure, our cards can be pretty wide if you have only 1 column. In that case the controls should be right aligned which I don't think they are now.